src/main/java/org/reaktivity/nukleus/function/CommandHandler.java

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright 2016-2017 The Reaktivity Project
+ *
+ * The Reaktivity Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package org.reaktivity.nukleus.function;
+
+import org.agrona.DirectBuffer;
+import org.agrona.MutableDirectBuffer;
+
+@FunctionalInterface
+public interface CommandHandler
+{
+    void handle(DirectBuffer buffer, int index, int length, MessageConsumer replyTo, MutableDirectBuffer replyBuffer);
```

Comment:
We'll need to address this later to keep the `replyBuffer` out of the method signature.